frontend/src/utils/sortAdmissions.ts

```diff
@@ -0,0 +1,66 @@
+import { Application } from "src/types";
+
+interface ApplicationComparator {
+  compare(app1: Application, app2: Application): number;
+  description: string;
+}
+
+const CreatedAtComparatorDesc: ApplicationComparator = {
+  compare: (app1, app2) => {
+    const date1 = new Date(app1.created_at);
+    const date2 = new Date(app2.created_at);
+    return date1.getTime() - date2.getTime();
+  },
+  description: "Dato Opprettet Synkende",
+};
+
+const CreatedAtComparatorAsc: ApplicationComparator = {
+  compare: (app1, app2) => {
+    const date1 = new Date(app1.created_at);
+    const date2 = new Date(app2.created_at);
+    return date2.getTime() - date1.getTime();
+  },
+  description: "Dato Opprettet Stigende",
+};
```

Comment:
Couldn't this just be the negation of the`CreatedAtComparatorDesc`? Then we wouldn't have as much duplicated code:) The same goes for all the other ones.